core/domain/voiceover_domain.py

```diff
@@ -204,7 +196,8 @@ def remove_voiceover(
         """Removes voiceover from the entity voiceover instance.
 
         Args:
-            content_id: str. The ID of the content.
+            content_id: str. The ID of the content for which the voiceover is
```

Comment:
This docstring is wrong. Do not just copy-paste.

(There are a lot of comments like this, which is really something that shouldn't be making it into PR reviews. Please proofread your work.)

Reply:
Updated now, I'm sorry this was missed earlier.